### DISCUSSION TITLE
latest version 59 (?) update problem

### DISCUSSION THREAD

**Discussion by userMak** [music-assistant/support#1388] [Feature requests and ideas]
Hi
I just updated MA and I am getting the following errors. Are you aware? 

Config entry 'Music Assistant' for mass integration not ready yet: Invalid server version: Schema version is incompatible: 23, the server requires at least 23 - update the Music Assistant client to a more recent version or downgrade the server.; Retrying in background

![image](https://github.com/music-assistant/hass-music-assistant/assets/55662933/0f4a51f4-9da3-4f17-9d75-2e825e3e99a5)

Music Assistant starts but the entities are gone

![image](https://github.com/music-assistant/hass-music-assistant/assets/55662933/a8824a1f-a807-431d-a35c-e06ac1a85172)

**Comment by marcelveldt** [comment]
So the answer is LITERALLY in the message... download a new version of the HACS integration...

**Comment by userMak** [comment]
Sorry, but I always forget that MA is in HACS also.

**Reply by marcelveldt** [reply]
Yes, once we're out of beta this kind of breaking changes will no longer happen

**Comment by userMak** [comment]
hmm
in the latest version I don't see the option to play and delete the queue. Is it me or something changed?

![image](https://github.com/music-assistant/hass-music-assistant/assets/55662933/10af1b64-7640-446e-9153-e0f8da594b72)

**Reply by marcelveldt** [reply]
Based on some discussions on discord it has become the default.

**Reply by userMak** [reply]
a sorry.. now I understand.. It clears by default the queue now??

edit: Just check it.. It clears the queue as it should! that's great

**Reply by OzGav** [reply]
Yes play now does but you also have play next and add to queue as options.

**Reply by userMak** [reply]
Yes it is fine now!!

Something I am missing or I don't now how to do it, is for example I play a playlist and I want to listen one song of the list that is already played.. How can i do it ? Obviously (at least in other players) when I choose a previous song the playlist is repeated from this song and after.. In MA I don't see this option

edit: just test it. If I choose the dong from queue it goes at the playlist with all the songs below it.. So it is fine that too!.

**Comment by userMak** [comment]
....
